spec/rails_helper.rb

```diff
@@ -8,6 +8,9 @@
 require 'steps/user_steps'
 require 'steps/approval_steps'
 
+# mimic production env for view+mail
+ENV["DISABLE_SANDBOX_WARNING"] = 'true'
```

Comment:
Minor: Hmmm...is it worth setting this explicitly for the tests that it affects?